ceph.spec.in

```diff
@@ -540,11 +540,13 @@ Requires:       python%{python3_pkgversion}-dateutil
 %if 0%{?fedora} || 0%{?rhel} >= 8
 Requires:       python%{python3_pkgversion}-cherrypy
 Requires:       python%{python3_pkgversion}-pyyaml
+Requires:       python%{python3_pkgversion}-typing-extensions
```

Comment:
if the package names are identical on all of these distros, why not extract them out?

Reply:
Unfortunately they're not exactly the same: note the dash vs. underscore in `typing[_-]extensions`.